R/init.R

```diff
@@ -223,6 +230,10 @@ init <- function(data,
   res <- list(
     ui = ui_teal_with_splash(id = id, data = data, title = title, header = header, footer = footer),
     server = function(input, output, session) {
+      if (length(landing) > 0L) {
```

Comment:
Now that we test for `length(landing) > 1L` above, this could be changed to `length(landing) == 1L` but I'm not convinced it's important.

Reply:
Frankly I think `length(landing) > 0L` means (allowing everything of lenght greater than 0), and `length(landing) == 1L` allows only the case of length 1, which is what we want and which is direct. Even though we checked for the length above. Will substitute

Reply:
c47552e35154db23259fdea90a5d9b282bc455fe